src/cmd/tools/read_data_files/main/main.go

```diff
@@ -248,20 +254,63 @@ func main() {
 	}
 }
 
+func getVolumeNumber(pathPrefix string, namespace string, shard int, blockStart string) (int, error) {
+	nsID := ident.StringID(namespace)
+	path := fs.NamespaceDataDirPath(pathPrefix, nsID)
+	files, err := ioutil.ReadDir(path + "/" + strconv.Itoa(shard))
+	if err != nil {
+		return 0, fmt.Errorf("failed reading data directory: %w", err)
+	}
+
+	volumeSet := make(map[int]struct{})
```

Comment:
Should delete it. It doesn't have any effect.

Reply:
fixed